apps/files/src/components/Collaborators/mixins.js

```diff
@@ -8,6 +38,33 @@ export default {
     $_ocCollaborators_switchPermission (permission) {
       this[permission] = !this[permission]
       this.editing = true
+      this.toggleCollaboratorsEdit(true)
+    },
+    $_ocCollaborators_loadAvatar (item) {
+      const dav = this.$client.helpers._davPath
+      const headers = new Headers()
+      const url = `${dav}/avatars/${item.value.shareWith}/128.png`
```

Comment:
image-source .... one day ...

Reply:
Not sure if image-source would help here. We will need to show some placeholder images in case the response is 404. AFAIK image-source doesn't handle this. And since it's avatar specific functionality not sure if we want to include it there.